### PR TITLE
[frontend/backend] Adding the correct version of RabbitMQ

### PR DESCRIPTION
**Pull request by Dimfacion**
### Proposed changes

* Adding a call to the management plugin of rabbitMQ to get the right version
* Adding a property to convey the right version to the frontend

### Related issues

* issue #841 
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

**Review by johanah29 — APPROVED** [review]

File: openbas-api/src/main/java/io/openbas/rest/settings/PlatformSettingsApi.java [hunk 1]
```diff
@@ -157,6 +159,39 @@ private Setting resolveFromMap(Map<String, Setting> dbSettings, String themeKey,
...
+
+            // Init the headers
+            HttpHeaders headers = new HttpHeaders();
+            headers.setAccept(Collections.singletonList(MediaType.APPLICATION_JSON));
+            headers.setBasicAuth(this.openBASConfig.getRabbitmqUser(), this.openBASConfig.getRabbitmqPass());
+            HttpEntity<String> entity = new HttpEntity<>("parameters", headers);
+
+            // Make the call
+            ResponseEntity<?> result =
+                    restTemplate.exchange(uri, HttpMethod.GET, entity, String.class);
```
Comment (RomuDeuxfois) [inline openbas-api/src/main/java/io/openbas/rest/settings/PlatformSettingsApi.java:185]: Can we move this logic to a dedicated service or client class for Rabbit ? 
What do you think ?

**Comment by RomuDeuxfois** [comment]
Tested 🆗

File: openbas-api/src/main/java/io/openbas/rest/settings/PlatformSettingsApi.java (hunk 1 above)
Comment (Dimfacion) [reply openbas-api/src/main/java/io/openbas/rest/settings/PlatformSettingsApi.java:185]: I was trying to reduce as much as possible the footprint of this code as this is not very elegant but that would make sense to have a dedicated class (helper ?) for this

Comment (RomuDeuxfois) [reply openbas-api/src/main/java/io/openbas/rest/settings/PlatformSettingsApi.java:185]: I vote for it, the code can be more readable/separated.

**Review by SamuelHassine — APPROVED** [review]

**Pull request merged**